src/index.js

```diff
@@ -606,11 +628,18 @@ class ReactTooltip extends React.Component {
 
   removeScrollListener(currentTarget) {
     const isCaptureMode = this.isCapture(currentTarget);
-    window.removeEventListener("scroll", this.hideTooltipOnScroll, isCaptureMode);
+    window.removeEventListener(
+      "scroll",
+      this.hideTooltipOnScroll,
+      isCaptureMode
+    );
   }
 
   // Calculation the position
   updatePosition() {
+    if (!this.tooltipRef) {
```

Comment:
This means we always need to pass a ref for this to work right?

Reply:
react-tooltip must have ref to be able to perform `getBoundingClientRect` on its node, otherwise we are trying to call `getBoundingClientRect` on undefined.